<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background: rgb(54, 64, 88);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="titleWidget" native="true">
        <property name="styleSheet">
         <string notr="true">background: rgb(46, 55, 76);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="titleIconLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="pixmap">
              <pixmap>:/resources/logo_title.png</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="titleLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
font-family: &quot;Helvetica&quot;, Helvetica, serif;
font-size: 17px;\n	font-weight: plain;</string>
             </property>
             <property name="text">
              <string>Installeur</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="closeButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>34</width>
               <height>34</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
background-color:rgb(46, 55, 76);
border: 0;}
QPushButton:hover {
background-color: rgb(33, 42, 63);
}</string>
             </property>
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="mainWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QWidget" name="dowloaderWidget" native="true">
             <property name="styleSheet">
              <string notr="true">background-color:#333;</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="1" column="0" colspan="2">
                 <widget class="QLabel" name="currentFileLabel">
                  <property name="styleSheet">
                   <string notr="true">color:white;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0" colspan="3">
                 <widget class="QProgressBar" name="totalProgressBar">
                  <property name="value">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" colspan="2">
                 <widget class="QLabel" name="serverLabel">
                  <property name="styleSheet">
                   <string notr="true">color:white;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QPushButton" name="pushButton">
                  <property name="text">
                   <string>PushButton</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" colspan="3">
                 <widget class="QProgressBar" name="singleProgressBar">
                  <property name="value">
                   <number>0</number>
                  </property>
                  <property name="textVisible">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="remainingTimeLabel">
                  <property name="styleSheet">
                   <string notr="true">color:#fff;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="3" column="2">
                 <widget class="QLabel" name="speedLabel">
                  <property name="styleSheet">
                   <string notr="true">color:white;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QWidget" name="downloaderWidget" native="true">
             <property name="styleSheet">
              <string notr="true">background-color:#333;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
